docs/managed_zones.md

```diff
@@ -0,0 +1,52 @@
+
+
+### Create the mctc namespace
+
+kubectl create ns mctc
+
+### Start the controller
+
+The controller must be started with the following AWS Env Vars:
+
+aws-credentials.env
+```
+AWS_ACCESS_KEY_ID=$MY_AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=$MY_AWS_SECRET_ACCESS_KEY
+AWS_REGION=us-east-1
+```
+
+```bash
+make build install && (export $(cat ./aws-credentials.env | xargs) && ./bin/manager --zap-log-level=3 --webhooks-port=0)
+```
+
+### Create a managed zone
+
+MCTC requires at least one Managed Zone to be created:
+
+dev.hcpapps.net-managedzone.yaml
+```yaml
+apiVersion: kuadrant.io/v1
+kind: ManagedZone
+metadata:
+  name: dev.hcpapps.net
+spec:
+  domainName: dev.hcpapps.net
+  description: "MY root zone for development!!"
+  id: MY_HOSTED_ZONE_ID
+```
+
+```bash
+kubectl apply -f dev.hcpapps.net-managedzone.yaml -n mctc
+```
+
+The `id` is the provider ID for an existing hosted zone in route53. Note, if you do not have a hosted zone, you can leave this blank and it will create one for `domainName` assuming the aws account has that domain registered in route53.
+
+```
+kubectl get managedzones -A
+NAMESPACE   NAME             DOMAIN NAME      ID                                  RECORD COUNT   NAMESERVERS                                                                                         READY
+mctc        dev.hcpapps.net  dev.hcpapps.net   /hostedzone/xy....zz               2              ["ns-2005.awsdns-58.co.uk","ns-627.awsdns-14.net","ns-1160.awsdns-17.org","ns-263.awsdns-32.com"]   True
+```
+
+### Create a managed zone
```

Comment:
looks left over?

Reply:
This was accidentally committed, will be completly rewritten